specification/ilm/explain_lifecycle/ExplainLifecycleRequest.ts

```diff
@@ -48,15 +47,5 @@ export interface Request extends RequestBase {
      * Filters the returned indices to only indices that are managed by ILM.
      */
     only_managed?: boolean
-    /**
-     * Period to wait for a connection to the master node. If no response is received before the timeout expires, the request fails and returns an error.
-     * @server_default 30s
-     */
-    master_timeout?: Duration
```

Comment:
master_timeout is supported: https://github.com/elastic/elasticsearch/blob/ce7a0c86fb975f76dfbea5eff1cf51602dbed184/x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/action/RestExplainLifecycleAction.java#L44